src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -41,6 +45,25 @@
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        StringBuilder sb = new StringBuilder("Отчёт за период " + dateFrom + " - " + dateTo + "\n");
+        final DateTimeFormatter F = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate dateFromDate = LocalDate.parse(dateFrom, F).minusDays(1);
```

Comment:
don't use `minusDays` and `plusDays`. these operations are redundant

Reply:
done